src/components/Input/Radio/story.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+
+import Radio from '.';
+import Readme from './README.md';
+
+import { InteractiveInput, Wrap } from '../../../stories/storybook-helpers';
+
+storiesOf('Planets/Input/Radio', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Wrap>
+        {/*
+          Eslint doesn't recognise that the `InteractiveInput` wrappers are
+          putting IDs on input elements, but they are.
+        */}
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+        <label className="text-xl inline-block mb-2">
```

Comment:
You might be able to remove the `eslint-disable` by adding `htmlFor="SampleRadio"`, since you're not using the `InteractiveInput` wrapper anymore.